tests/spark/test_mrjob_spark_harness.py

```diff
@@ -198,5 +215,24 @@ def test_passthru_args(self):
             input_bytes=input_bytes,
             extra_args=['--chars', '--ignore', 'to'])
 
+    def test_combiner_called(self):
```

Comment:
This doesn't really belong in this test class, which is about comparing output between runs with and without the harness. I wouldn't worry about the ~5s overhead of creating another `SingleSparkContext` (and I could optimize it out anyhow).

Reply:
I take that back; there's already another test that just uses `self._harness_job()`. This is fine where it is.